### DISCUSSION TITLE
Map display issue

### DISCUSSION THREAD

**Discussion by paveler** [uber/react-map-gl#1602] [Q&A]
My `react-map-gl` component does not show the map. The css styling seems to be loaded, as soon as the background is dark:

![image](https://api.alfa.directual.com/fileUploaded/geo/d7d398df-a178-408b-bcc0-fdf189f8bd29.jpg)

It happens in [my react-project](https://github.com/directual/directual-web-components/blob/master/src/components/fps/dataentry/map/map.js)

The same code in [codesandbox](https://codesandbox.io/s/headless-bush-4i2ui?file=/src/App.js:0-931) works well!

What could be the issue? Why the same code does not work?

**Comment by Cmoen11** [comment]
One of the common issues is when you build your project. 
You should get an error in the console if this is the case.
https://github.com/visgl/react-map-gl/issues/1266

There are serval different solutions you can try within the issue I've referenced.
